### PR TITLE
Speed up of selu

### PR DESCRIPTION
**Pull request by racinmat**
Implementing speedup of selu based on this comment: https://github.com/FluxML/Zygote.jl/pull/367#issuecomment-541844988
Utilizing vectorized ifelse based on https://github.com/FluxML/NNlib.jl/blob/master/src/activation.jl#L159.

### CODE REVIEW

File: src/lib/nnlib.jl [hunk 1]
```diff
@@ -1,12 +1,23 @@
 using NNlib
 import NNlib: softmax, ∇softmax, logsoftmax, ∇logsoftmax, conv, depthwiseconv, ∇conv_data, ∇depthwiseconv_data, maxpool, meanpool, σ, relu, batched_mul, batched_adjoint
+import LoopVectorization: vifelse
```
Comment (DhairyaLGandhi) [inline src/lib/nnlib.jl:3]: Based on current discussion, vectorisation would need some changes to how we plan to integrate lv within lv. Could we remove the dep from this PR?

Comment (racinmat) [reply src/lib/nnlib.jl:3]: Oh, my bad, sure, I'll return it.

**Comment by DhairyaLGandhi** [comment]
Thank you so much!

Would you mind adding the tanh adjoint in there as well?

**Comment by racinmat** [comment]
Done.

**Comment by DhairyaLGandhi** [comment]
Thanks!

**Comment by racinmat** [comment]
May I expect a new patch version released, so we could start using it?

**Pull request merged**